package.json

```diff
@@ -50,6 +50,8 @@
   },
   "license": "MIT",
   "dependencies": {
+    "@azure/core-auth": "1.1.4",
+    "@azure/ms-rest-nodeauth": "^3.0.6",
```

Comment:
Looks like we need this dependency only for the type `TokenResponse`. Can we make a copy of this instead? Otherwise this heavy dependency will get pulled in unnecessarily in user's code.

Reply:
Agreed

Reply:
Fixed.